src/models/babyAnimals.js

```diff
@@ -1,12 +1,12 @@
-// 'use strict';
+'use strict';
 
-// class BabyAnimal {
-//   constructor(name){
-//     this.id = 0;
-//     this.name = name;
-//     this.db = [];
-//   }
+const mongoose = require('mongoose');
 
+const babyAnimalSchema = mongoose.Schema({
+  name: { type: String, require: true },
+  type: { type: String, require: true, enum: ['PIGLET, KITTEN, PUPPY, CHICK, JOEY, CUB']},
```

Comment:
When I try and add something to my local database, I am having an issue with the enum on animals and flowers